specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json

```diff
@@ -788,6 +998,123 @@
+    },
+    "VirtualNetworkPeering": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/VirtualNetworkPeeringPropertiesFormat",
+          "description": "List of properties for vNet Peering"
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the virtual network peering resource",
+          "readOnly": true
+        },
+        "id": {
+          "type": "string",
+          "description": "Resource ID.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "type of the virtual network peering resource",
+          "readOnly": true
+        }
+      },
+      "description": "Peerings in a VirtualNetwork resource",
+      "x-ms-azure-resource": true
+    },
+    "VirtualNetworkPeeringPropertiesFormat": {
+      "properties": {
+        "allowVirtualNetworkAccess": {
+          "type": "boolean",
+          "description": "Whether the VMs in the local virtual network space would be able to access the VMs in remote virtual network space."
+        },
+        "allowForwardedTraffic": {
+          "type": "boolean",
+          "description": "Whether the forwarded traffic from the VMs in the local virtual network will be allowed/disallowed in remote virtual network."
+        },
+        "allowGatewayTransit": {
+          "type": "boolean",
+          "description": "If gateway links can be used in remote virtual networking to link to this virtual network."
+        },
+        "useRemoteGateways": {
+          "type": "boolean",
+          "description": "If remote gateways can be used on this virtual network. If the flag is set to true, and allowGatewayTransit on remote peering is also true, virtual network will use gateways of remote virtual network for transit. Only one peering can have this flag set to true. This flag cannot be set if virtual network already has a gateway."
+        },
+        "databricksVirtualNetwork": {
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "The Id of the databricks virtual network."
+            }
+          },
+          "description": " The remote virtual network should be in the same region. See here to learn more (https://docs.microsoft.com/en-us/azure/databricks/administration-guide/cloud-configurations/azure/vnet-peering)."
+        },
+        "databricksAddressSpace": {
+          "$ref": "#/definitions/AddressSpace",
+          "description": "The reference to the databricks virtual network address space."
+        },
+        "remoteVirtualNetwork": {
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "The Id of the remote virtual network."
+            }
+          },
+          "required": true,
```

Comment:
>"required": true, [](start = 10, length = 17)

required is defined at a higher level than this.  See the Workspace definition for an example of how to signify required properties